### DISCUSSION TITLE
SQL query based on Option Picker

### DISCUSSION THREAD

**Discussion by rcusters1985** [Budibase/budibase#11605] [Help]
Hi, 

There must be a very simple explanation, but for some reasons I can't figured it out:

I have the following SQL query:

`SELECT SUM(Bedrag) FROM Uitgaven WHERE Maand = {{Maand}}`

What I would like to perform is reference {{Maand}} to the options that are selected in the option picker, then show the result of that query in a Static card for example. But for some reason I cannot setup the reference {{Maand}} in the query settings. (Also when I tried to setup a binding), maybe because its Sandboxed?

So in short, When I select a month in the dropdown list I would like to run the query that sums up the amount of money of that selected month.

Thanks!

**Comment by zedascouves** [comment]
I think I had a similar issue. Have you tried calling the variable {{Maand}} something like {{InputMaand}}? I think the platform gets confused with variables that have the same name as the columns.
